docs/Using Fleet/enroll-hosts.md

```diff
@@ -141,6 +159,7 @@ The following command-line flags allow you to configure an osquery installer fur
 | --enable-scripts           | Enable script execution (default: `false`)                                                                                              |
 | --debug                    | Enable debug logging (default: `false`)                                                                                                 |
 | --verbose                  | Log detailed information when building the package (default: false)                                                                     |
+| --local-wix-dir            | Use local install of WiX instead of Docker Hub (only available on Windows w/ WiX v3)                                                    |
```

Comment:
Let's clarify that we it expects `heat.exe`, `light.exe` and `candle.exe` to be there.